csm/core/controllers/view.py

```diff
@@ -35,7 +35,8 @@ class CsmAuth:
     ATTR_PERMISSIONS = '_csm_auth_permissions_'
     # ADD all hybrid api's here with required keys, use hybrid decorator
     # TODO: make it dynamic by adding required key as param to hybrid decorator.
-    HYBRID_APIS = {"GET:/api/v2/metrics/stats/perf": const.AUTH}
+    HYBRID_APIS = {"GET:/api/v2/metrics/stats/perf": const.AUTH,
+                   "GET:/api/v2/capacity/status/bytecount": const.AUTH}
```

Comment:
Do not add bytecount API here. We have not informed QA about it. Test it thoroughly.

This change will make bytecount api public

Reply:
Removed bytecount api from hybrid api's. Will add it for testing